src/main/java/com/blakebr0/extendedcrafting/compat/jei/combinationcrafting/CombinationCraftingWrapper.java

```diff
@@ -28,11 +28,11 @@ public class CombinationCraftingWrapper implements IRecipeWrapper {
 	public CombinationCraftingWrapper(IJeiHelpers helpers, CombinationRecipe recipe) {
 		this.helpers = helpers;
 		this.recipe = recipe;
-		int period = recipe.getPedestalIngredients()
+		int period = Math.max(1, recipe.getPedestalIngredients()
 				.stream()
 				.map(Ingredient::getMatchingStacks)
 				.map(a -> a.length)
-				.reduce(1, (a, b) -> a * b);
+				.reduce(1, (a, b) -> a * b));
 		timer = helpers.getGuiHelper()
 				.createTickTimer(period * 20, period, false);
```

Comment:
It might be better to put the clamping around `a.length` instead, that way one empty ingredient list won't stop other ingredients from cycling.

Reply:
More subtly, this doesn't include the period from the core input ingredient (which you've now added to the ingredient list), so the period might be wrong for it.